modules/tinymce/src/core/main/ts/fmt/ExpandRange.ts

```diff
@@ -228,7 +238,10 @@ const findParentContainer = (
 
 const isSelfOrParentBookmark = (container: Node) => isBookmarkNode(container.parentNode) || isBookmarkNode(container);
 
-const expandRng = (dom: DOMUtils, rng: Range, formatList: Format[], includeTrailingSpace: boolean = false): RangeLikeObject => {
+const expandRng = (dom: DOMUtils, rng: Range, formatList: Format[], expandOptions: ExpandOptions = {}): RangeLikeObject => {
+  const { includeTrailingSpace = false, expandToBlock = true } = expandOptions;
```

Comment:
```suggestion
const expandRng = (dom: DOMUtils, rng: Range, formatList: Format[], {includeTrailingSpace, expandToBlock}: ExpandOptions = { includeTrailingSpace = false, expandToBlock = true }): RangeLikeObject => {```
I'm not sure about the readability but maybe we can move the default in the function sign instead of have a line to extract the values with the dafaults

Reply:
Right but that makes the line longer. I think the original is better.